packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+/* eslint-disable no-undef */
```

Comment:
A more specific eslint ignore:
```suggestion
/* global globalThis */
```